src/xtgeo/grid3d/grid_properties.py

```diff
@@ -3,7 +3,17 @@
 
 import hashlib
 import warnings
-from typing import TYPE_CHECKING, List, Literal, Optional, Tuple, Union
+from typing import (
+    TYPE_CHECKING,
+    Any,
+    Iterable,
+    Iterator,
```

Comment:
I think `pyupgrade` would say these should come from `collections.abc` instead. I doubt this matters for type checking but consistency is nice